sql/incoming-schema-changes.sql

```diff
@@ -87,6 +87,7 @@ CREATE TABLE playertimes (
   userid VARCHAR(32) NOT NULL,
   time_in_minutes INT(5) unsigned not null,
   time_note VARCHAR(150),
+  createdate DATE DEFAULT (CURRENT_DATE),
```

Comment:
(Sorry, I thought I'd posted about this before, but I guess I didn't.)

```suggestion
  createdate DATETIME DEFAULT now(),
```

> https://stackoverflow.com/questions/31761047/what-difference-between-the-date-time-datetime-and-timestamp-types
> 
> **DATE:** It is used for values with a date part but no time part. MySQL retrieves and displays DATE values in ***YYYY-MM-DD*** format. The supported range is `1000-01-01` to `9999-12-31`.
> 
> **DATETIME:** It is used for values that contain both date and time parts. MySQL retrieves and displays DATETIME values in ***YYYY-MM-DD HH:MM:SS*** format. The supported range is `1000-01-01 00:00:00` to `9999-12-31 23:59:59`.
> 
> **TIMESTAMP:** It is also used for values that contain both date and time parts, and includes the time zone. TIMESTAMP has a range of `1970-01-01 00:00:01` UTC to `2038-01-19 03:14:07` UTC.

Reply:
What would we use the time for?

Reply:
Sorting, at least. I would have used it for the deanonymization thingy.

Reply:
Done